src/dc_e2ee.rs

```diff
@@ -1154,4 +1142,47 @@ Sent with my Delta Chat Messenger: https://delta.chat";
 
         unsafe { free(decrypted_mime as *mut _) };
     }
+
+    mod load_or_generate_self_public_key {
+        use super::*;
+
+        #[test]
+        fn test_existing() {
+            let t = dummy_context();
+            let addr = configure_alice_keypair(&t.ctx);
+            let key = load_or_generate_self_public_key(&t.ctx, addr);
+            assert!(key.is_ok());
+        }
+
+        #[test]
+        #[ignore] // generating keys is expensive
```

Comment:
it's only expensive in debug mode btw.

Reply:
oh, didn't know it was much better.  though tests often get run in debug mode during development, I don't think there's any harm in excluding some more tests in the default set.  CI will still run them and when you're working on an area where they matter you can run them too.

Reply:
sure